compute_tools/src/migration.rs

```diff
@@ -57,43 +57,44 @@ impl<'m> MigrationRunner<'m> {
     pub fn run_migrations(mut self) -> Result<()> {
         self.prepare_migrations()?;
 
-        let mut current_migration: usize = self.get_migration_id()? as usize;
-        let starting_migration_id = current_migration;
-
-        let query = "BEGIN";
```

Comment:
Hm, I think we need to move it inside `while`, not just drop, right? Otherwise, you have commit at the end, but no begin. Looks like now migrations are not executed withing an explicit transaction block?

Reply:
This looks like I failed on a rebase. Thanks for catching this!